format/spec.md

```diff
@@ -373,6 +378,13 @@ Notes:
 3. Strings are truncated to a valid UTF-8 string with no more than `L` code points.
 4. In contrast to strings, binary values do not have an assumed encoding and are truncated to `L` bytes.
 
+#### XZ2 Transform Details
+
+XZ2 is based on the paper [XZ-Ordering: A Space-Filling Curve for Objects with Spatial Extensions].
+
+Notes:
+1. Resolution must be a positive integer.  Defaults to TODO
```

Comment:
@jiayuasu do you have any suggestion for default here?

Reply:
GeoMesa has 12 as the default: https://github.com/locationtech/geomesa/blob/geomesa-4.0.3/geomesa-z3/src/main/scala/org/locationtech/geomesa/curve/XZSFC.scala#L13

Reply:
12 sounds fine. CC @Kontinuation

Reply:
GeoMesa uses a high XZ2 resolution when working with key-value stores such as Accumulo and HBase, it is not appropriate to always use a resolution that high for partitioning data (for instance, [GeoMesa on FileSystems](https://www.geomesa.org/documentation/stable/user/filesystem/partition_schemes.html)).

XZ2 resolution 11~12 works for city-scale data, but will generate too many partitions for country-scale or world-scale data. I'd like to have a smaller default value such as 7 to be safe on various kinds of data.

[![image](https://github.com/user-attachments/assets/c427bd18-751f-4503-b056-0c6a9dcf8317)](https://www.researchgate.net/publication/376632924_GeoLake_Bringing_Geospatial_Support_to_Lakehouses)